lib/api.spec.js

```diff
@@ -404,7 +403,7 @@ describe('AnxApi', function() {
 						done();
 					},
 				});
-				api.get('user');
+				api.get('user').catch((error) => error);
```

Comment:
Native promises throw `UnhandledPromiseRejectionWarning:` for these cases.

Reply:
Won't this swallow exceptions?

Reply:
Yeah; it was silently swallowing them before, also, though.

I think for these tests it's okay if the API requests fail, which is why there wasn't handling for errors, it's just validating some of the parameters / options that get generated by the API code.